# Exercise 4
  
___  

__Date Started__ : Nov 3, 2018

___

Content Details:
* Population Dynamics 
  * Analyzing Internet Statistics 
  
___

Data Used:
 1. [AnonymityPoll.csv](./AnonymityPoll.csv)

In [2]:
#loading data into frames


user_data = read.csv('AnonymityPoll.csv')

In [4]:
str(user_data)
summary(user_data)

'data.frame':	1002 obs. of  13 variables:
 $ Internet.Use          : int  1 1 0 1 0 1 1 0 0 1 ...
 $ Smartphone            : int  0 0 1 0 NA 1 0 0 NA 0 ...
 $ Sex                   : Factor w/ 2 levels "Female","Male": 2 2 1 2 1 2 1 1 2 1 ...
 $ Age                   : int  62 45 70 70 80 49 52 76 75 76 ...
 $ State                 : Factor w/ 49 levels "Alabama","Arizona",..: 20 39 29 10 10 41 21 31 32 32 ...
 $ Region                : Factor w/ 4 levels "Midwest","Northeast",..: 2 3 2 3 3 3 1 2 3 3 ...
 $ Conservativeness      : int  4 1 4 4 4 4 3 3 4 4 ...
 $ Info.On.Internet      : int  0 1 0 3 NA 6 3 NA NA 0 ...
 $ Worry.About.Info      : int  1 0 0 1 NA 0 1 NA NA 0 ...
 $ Privacy.Importance    : num  100 0 NA 88.9 NA ...
 $ Anonymity.Possible    : int  0 1 0 1 NA 1 0 NA NA 1 ...
 $ Tried.Masking.Identity: int  0 0 0 0 NA 1 0 NA NA 0 ...
 $ Privacy.Laws.Effective: int  0 1 NA 0 NA 0 1 NA 0 1 ...


  Internet.Use      Smartphone         Sex           Age       
 Min.   :0.0000   Min.   :0.0000   Female:505   Min.   :18.00  
 1st Qu.:1.0000   1st Qu.:0.0000   Male  :497   1st Qu.:37.00  
 Median :1.0000   Median :1.0000                Median :55.00  
 Mean   :0.7742   Mean   :0.5078                Mean   :52.37  
 3rd Qu.:1.0000   3rd Qu.:1.0000                3rd Qu.:66.00  
 Max.   :1.0000   Max.   :1.0000                Max.   :96.00  
 NA's   :1        NA's   :43                    NA's   :27     
          State           Region    Conservativeness Info.On.Internet
 California  :103   Midwest  :239   Min.   :1.000    Min.   : 0.000  
 Texas       : 72   Northeast:166   1st Qu.:3.000    1st Qu.: 2.000  
 New York    : 60   South    :359   Median :3.000    Median : 4.000  
 Pennsylvania: 45   West     :238   Mean   :3.277    Mean   : 3.795  
 Florida     : 42                   3rd Qu.:4.000    3rd Qu.: 6.000  
 Ohio        : 38                   Max.   :5.000    Max.   :11.000 

#### Solution to problem 1.1: 1002 user participated in the survey

In [14]:
# solution to problem 2
table(user_data$Smartphone)
table(is.na(user_data$Smartphone))


  0   1 
472 487 


FALSE  TRUE 
  959    43 

##### Solution to problem 1.2: 472 user use phone, 487 dont and there are 43 data NA in the sample study

In [32]:
#solution to determine the states in Midwest region
unique(user_data$State[user_data$Region == 'Midwest'])
#states in south region with maximum number of interviewee
sort(table(user_data$State[user_data$Region == 'South']), decreasing = TRUE)[1]

[1] Michigan     Ohio         Indiana      Minnesota    North Dakota
 [6] Wisconsin    Missouri     Illinois     Nebraska     South Dakota
[11] Kansas       Iowa        
49 Levels: Alabama Arizona Arkansas California Colorado ... Wyoming

Texas 
   72